webdriver/support/fixtures.py

```diff
@@ -115,3 +117,83 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def switch_to_inactive(session):
+    """Create a new browsing context, switch to that new context, and then
+    delete the context."""
+    def switch_to_inactive():
+        initial = session.send_session_command("GET", "window/handles")
+
+        session.send_session_command("POST",
+                                     "execute/sync",
+                                     dict(script="window.open();", args=[]))
```

Comment:
Why are you using `session.send_session_command` here when you can use `session.execute`?